_sources/rust-2019-rethinking-the-team-structure.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Rust 2019: Rethinking The Team Structure"
+---
+
+# Rust 2019 (8): Rethinking The Team Structure
+
+Rusts current team structure is mainly ad-hoc and built towards independence. If there's a group of people interested in the same thing, they get a label. That makes them addressable and shows that something is being worked on. Working groups are similar, but with a narrower scope. All of these are still rather ad-hoc, still. Usually, groups with a team-lead have formed.
+
+What I believe has worked really well for the project up until now. It's a process that's easy to agree on and quick to implement. Also, there's a fundamental agreement of trust between groups. Rust core has done a good job and fostering that.
+
+I don't think it's going to carry us to the next level. It believe constrains us with regards to bandwidth _and_ participation. A couple of ideas arouund that subject:
+
+## Single leaders
+
+I think it's time to move beyond single leaders. It's odd that we build our teams towards independence, but the teams themselves are hierarchical. The community team is currently experimenting with floating roles and experiences look good. It has multiple advantages: it shifts away responsible from full-time Rust employees, while still being flexible enough to have hobbyists choose their own pace and effort. It also makes the team resilient against sudden changes. I'd like to specifically mention [Ashley Williams](TODO: Twitter link) for starting the change towards that model.
+
+## Defined goals, roles and responsibilities better
+
+For most teams and W, the written down description and mission doesn't go beyond their name. This may not sound like a huge problem, but it makes it hard to evaluate progress. While we're doing public roadmapping on a project level, public roadmapping on a team level might make sense.
```

Comment:
whats "W" ?

Reply:
working groups, I'll fix that, thanks :)

Reply:
```suggestion
For most teams and WGs, the written down description and mission doesn't go beyond their name. This may not sound like a huge problem, but it makes it hard to evaluate progress. While we're doing public roadmapping on a project level, public roadmapping on a team level might make sense.
```